index.css

```diff
@@ -14,6 +14,13 @@
 .bg-regal-blue-dark {
   background-color: #000022;
 }
+.text-regal-blue {
+  color: #2a2840;
+}
+
+.text-regal-blue-dark {
+  color: #000022;
+}
```

Comment:
add variable name for colors

Reply:
okay will make these changes , thanks for reviewing the code 😄